data/repositories.toml

```diff
@@ -810,5 +810,6 @@ repositories = [
   'github.com/zulip/zulip',
   'github.com/ZupIT/charlescd',
   'github.com/ZupIT/ritchie-cli',
-  'github.com/ZupIT/ritchie-formulas'
+  'github.com/ZupIT/ritchie-formulas',
+  'https://github.com/DoonOnthon/GamePulse'
```

Comment:
Can you add this in the correct format and in the correct lexicographic order? Please refer to the other entries in this file.